### PR TITLE
Node Observability Operator: enable agent image mirroring to quay.io

### PR DESCRIPTION
**Pull request by alebedev87**

### CODE REVIEW

File: clusters/app.ci/registry-access/node-observability-operator/admin_node_observability_operator.yaml
Comment (hongkailiu) [inline clusters/app.ci/registry-access/node-observability-operator/admin_node_observability_operator.yaml:28]: Please user Rover group for RBACs.
https://docs.ci.openshift.org/docs/how-tos/rbac/

File: clusters/app.ci/registry-access/node-observability-operator/admin_node_observability_operator.yaml
Comment (hongkailiu) [inline clusters/app.ci/registry-access/node-observability-operator/admin_node_observability_operator.yaml:7]: registry.ci.openshift.org/ocp/4.11:node-observability-agent

If the image is promoted to ocp, why do we need the namespace?

File: core-services/image-mirroring/node-observability-operator/mapping_node_observability_operator_4_11
Comment (hongkailiu) [inline core-services/image-mirroring/node-observability-operator/mapping_node_observability_operator_4_11:1]: You need to contact the admin of `quay.io/node-observability-operator` for the credentials to push.

File: clusters/app.ci/registry-access/node-observability-operator/admin_node_observability_operator.yaml
Comment (alebedev87) [reply clusters/app.ci/registry-access/node-observability-operator/admin_node_observability_operator.yaml:7]: Created a new organization in quay.io for Node Observability Operator. Changed the promotion of the agent image from OCP to the new organization.

File: core-services/image-mirroring/node-observability-operator/mapping_node_observability_operator_4_11
Comment (alebedev87) [reply core-services/image-mirroring/node-observability-operator/mapping_node_observability_operator_4_11]: Changed the promotion target from OCP to this operator's dedicated ImageStream.

File: clusters/app.ci/registry-access/node-observability-operator/admin_node_observability_operator.yaml
Comment (alebedev87) [reply clusters/app.ci/registry-access/node-observability-operator/admin_node_observability_operator.yaml]: Created a Rover group, should I do any configuration for `app.ci` cluster or it'll be synced automatically?

**Comment by droslean** [comment]
@alebedev87 What is the status of this PR? Also, can your add more context on why do you need to create a handcrafted job to mirror those images in quay? We already have this automation.

**Comment by alebedev87** [comment]
@droslean : the status is "on hold", we will come back to it once the 4.11 release will be cut off. It's job to mirror to a new organization in quay.io (different from `openshift`). You can see [the CI docs](https://docs.ci.openshift.org/docs/how-tos/mirroring-to-quay/#configuring-mirroring-for-new-organization) which instructs the new job for the new organization.

**Comment by alebedev87** [comment]
@hongkailiu : I came back to the PR: fixed the conflicts and removed the hold label. Can you please have another look?

**Comment by alebedev87** [comment]
@droslean : I came back to this PR, rebased it and unheld. Can you please have another look?

File: ci-operator/jobs/infra-image-mirroring.yaml
Comment (droslean) [inline ci-operator/jobs/infra-image-mirroring.yaml:1252]: How important this is it to run every hour?

Comment (alebedev87) [reply ci-operator/jobs/infra-image-mirroring.yaml:1252]: Not very critical. However the idea was that we can be quicker than we used to be with the mirroring into the openshift organization when it took 4,5,6 hours (OCP release and test).
The time gained here would allow us to use the operand fixes in the operator repository faster. This comes quite touchy when some Bugs needs to be fixed fast or when the release deadline is coming.

**Comment by alebedev87** [comment]
@droslean: Do you have any other remarks?

**Review by droslean — APPROVED** [review]

**Comment by alebedev87** [comment]
@hongkailiu : can you please have anything look or approve if all is fine with you?

**Comment by alebedev87** [comment]
/approve

works?

**Pull request merged**
